src/components/v5/common/ActionSidebar/partials/forms/MintTokenForm/hooks.ts

```diff
@@ -28,11 +28,9 @@ export const useMintToken = (
     defaultValues: useMemo<DeepPartial<MintTokenFormValues>>(
       () => ({
         createdIn: Id.RootDomain.toString(),
-        amount: {
-          tokenAddress: colony?.nativeToken.tokenAddress,
-        },
+        amount: {},
       }),
-      [colony?.nativeToken.tokenAddress],
+      [],
```

Comment:
If we are assuming that the amount will correspond to the native token of the colony, then this field doesn't need to be an object like the validation suggests:
```
amount: object()
  .shape({
    amount: number()
      .required(() => 'required field')
      .transform((value) => toFinite(value))
      .moreThan(0, () => 'Amount must be greater than zero.'),
  })
  .required(),
```

 This could be just, which is more or less what we do already in the old mint tokens form:
 ```
 amount: number()
  .required(() => 'required field')
  .transform((value) => toFinite(value))
  .moreThan(0, () => 'Amount must be greater than zero.'),
 ```

Unless I'm missing something

Reply:
@ArmandoGraterol I agree with you, it would be better to have a different structure here, but I can't make such change, because we are using `AmountField` component here which is used also in other forms and it contains 2 fields inside: one for `amount` and one for `tokenAddress`, so it has to be an object. 
To make the change requested by you, I would have to create a new component (which will cause code duplication) or modify this one (it will require a lot of changes in the code of other forms in this case).